open-company/open-company-web
# Section delete notability

Card: https://trello.com/c/3ZgEVoNR

To test:
- go to a section
- click cog icon
- [x] do you see the delete section button in carrot red? Good
- delete a section
- [x] did you get the notification once delete is done? Good
